app/scripts/controllers/projects.js

```diff
@@ -16,13 +16,37 @@ angular.module('openshiftConsole')
     $scope.showGetStarted = false;
     $scope.canCreate = undefined;
 
+    var sortProjects = function() {
+      var sortID = _.get($scope, 'sortConfig.currentField.id', 'metadata.name');
+
+      $scope.projects = _.sortByOrder($scope.projects, function(project) {
+        return _.get(project, sortID).toLowerCase() || _.get(project, 'metadata.name').toLowerCase();
```

Comment:
Shouldn't this be...?

```js
 _.get(project, sortID, '').toLowerCase()
```

Otherwise it will blow up if the project doesn't have a value for `sortID`